### PR TITLE
Curator 4.1

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/pull/14594

### CODE REVIEW

**Comment by mburke5678** [comment]
@josefkarasek Not sure I am doing this right.

File: modules/efk-logging-curator-actions.adoc [hunk 1]
```diff
@@ -5,19 +5,6 @@
 [id="efk-logging-curator-actions-{context}"]
```
Comment (josefkarasek) [inline modules/efk-logging-curator-actions.adoc:5]: Let's use text from https://github.com/openshift/openshift-docs/pull/14526/files#diff-4cc0705fa09f2587e3127fbecc751f87 here?

File: modules/efk-logging-curator-troubleshoot.adoc [hunk 2]
```diff
@@ -6,23 +6,22 @@
 = Troubleshooting Curator
 
 You can use information in this section for debugging Curator. For example, if curator is in failed state, but the log messages do not
-provide a reason, you could increase the log level and trigger a new job, instead of waiting for another scheduled run of the cron job.
+provide a reason, you could increase the log level and trugger a new job, instead of waiting for another scheduled run of the cron job.
```
Comment (josefkarasek) [inline modules/efk-logging-curator-troubleshoot.adoc]: `trigger` vs `trugger` ?

File: modules/efk-logging-curator-scripted.adoc [hunk 3]
```diff
@@ -39,18 +40,22 @@ $ oc create configmap curator \
...
+
+.. Copy the Curator configuration file and the *action* file
 from the Curator configuration map and create separate files for each:
 +
 ----
 $ oc extract configmap/curator --keys=curator5.yaml,actions.yaml --to=/my/config
 ----
 +
-.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/actions.yaml_* files.
+.. Edit the *_/my/config/curator5.yaml_* and *_/my/config/action.yaml_* files.
```
Comment (josefkarasek) [inline modules/efk-logging-curator-scripted.adoc]: When I wrote the scripts that generate the action file I was still calling it _"actions"_ file - when referring to the file as a path somewhere on a computer, it is still `actions.yaml`.

File: modules/efk-logging-curator-troubleshoot.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/efk-logging-curator-troubleshoot.adoc]: Not sure how that happened!

**Pull request merged**